Sumangala1.c

```diff
@@ -0,0 +1,11 @@
+//program to find the sum inclusive of the start and the end no. mentioned
+#include<stdio.h>
+int main()
+{
+int n1,n2,i,sum=0;
+scanf("%d %d",&n1,&n2);
+for(i=n1;i<=n2;i++)
```

Comment:
Make each of these a block and comment for each of the block what it does.

Reply:
ok